src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -63,33 +89,42 @@ static ColliderCylinderInit sCylinderInit = {
     { 30, 40, 0, { 0, 0, 0 } },
 };
 
-void func_80B13AA0(EnTa* this, EnTaActionFunc arg1, EnTaUnkFunc arg2) {
+void EnTa_SetActionFunctions(EnTa* this, EnTaActionFunc arg1, EnTaBlinkFunc arg2) {
```

Comment:
In general this kind of function we call `_SetupAction` (see the many others in the repo), though this one is different since it also has that blink function, but I think it's still fine
```suggestion
void EnTa_SetupAction(EnTa* this, EnTaActionFunc arg1, EnTaBlinkFunc arg2) {
```

Oh also `EnTaBlinkFunc` is the anim func actually?

Reply:
I had the anim and blink function types mixed up in some places, I noticed this earlier but I guess I forgot to push the fixes 😄